all_os_install_requirements.sh

```diff
@@ -0,0 +1,47 @@
+if ! which minikube 2>/dev/null ; then
```

Comment:
what about putting this in 01_install_requirements instead of creating a new file ?